src/coreclr/vm/codeman.cpp

```diff
@@ -42,6 +45,13 @@
 #include "perfmap.h"
 #endif
 
+#if defined(TARGET_LINUX) && !defined(PR_SVE_GET_VL)
+#define PR_SVE_SET_VL           50	/* set task vector length */
+#define PR_SVE_GET_VL           51	/* get task vector length */
+#define PR_SVE_VL_LEN_MASK      0xffff
+#define PR_SVE_VL_INHERIT       (1 << 17) /* inherit across exec */
```

Comment:
Defining these vars explicitly as there are some builds failing, such as [this](https://dev.azure.com/dnceng-public/public/_build/results?buildId=649553&view=logs&j=e2af9a1a-ae86-5313-2146-b7306ad4db64&t=8f10af5c-d1dd-5c33-de37-3925c5c2a252&l=2740).
Kindly suggest if there is a better way of doing this.